app/models/container_project.rb

```diff
@@ -10,22 +10,22 @@ class ContainerProject < ApplicationRecord
   include Purging
   belongs_to :ext_management_system, :class_name => "ManageIQ::Providers::ContainerManager", :foreign_key => "ems_id", :inverse_of => :container_projects
   has_many :container_groups, -> { active }, :inverse_of => :container_project
-  has_many :container_routes
-  has_many :container_replicators
-  has_many :container_services
+  has_many :container_routes      # delete to be handled by refresh
+  has_many :container_replicators # delete to be handled by refresh
+  has_many :container_services   # delete to be handled by refresh
```

Comment:
😉 

```suggestion
  has_many :container_services    # delete to be handled by refresh
```

Reply:
haha, git history now makes this look like it never happened.  🪄